playground/deepspeed_config_s2.json

```diff
@@ -1,12 +1,34 @@
 {
-  "zero_optimization": {
-    "stage": 2,
-    "offload_optimizer": {
-      "device": "cpu"
+  "bf16": {
+    "enabled": "auto"
+  },
+  "optimizer": {
+    "type": "AdamW",
+    "params": {
+      "lr": "auto",
+      "betas": [
+        0.9,
+        0.999
+      ],
+      "eps": "auto",
+      "weight_decay": "auto"
     },
-    "contiguous_gradients": true,
-    "overlap_comm": true
+    "zero_optimization": {
+      "stage": 2,
+      "offload_optimizer": {
+        "device": "cpu",
+        "pin_memory": true
+      },
+      "allgather_partitions": true,
+      "allgather_bucket_size": 2e8,
+      "reduce_scatter": true,
+      "reduce_bucket_size": 2e8,
+      "overlap_comm": true,
+      "contiguous_gradients": true
+    }
   },
+  "train_batch_size": "auto",
   "train_micro_batch_size_per_gpu": "auto",
-  "gradient_accumulation_steps": "auto"
+  "gradient_accumulation_steps": "auto",
+  "zero_force_ds_cpu_optimizer": false
```

Comment:
Is there a reason we aren't using the deepspeed's optimized Adam?

Reply:
I have reset the config to what is on your repo.